src/awesome_map/BoundaryInterceptor.js

```diff
@@ -151,6 +163,28 @@ define(function(require) {
         this._easing = options.easing || EasingFunctions.easeOutQuart;
 
         /**
+         * This value defines the minimum number of pixels beyond a boundary a scroll event
+         * must reach for boundary events to be fired, so that light touches and small scrolls 
+         * (by default) wont fire events constantly.  A value of 0 will cause every event which
+         * passes a boundary to fire a boundary event.
+         * @type {number}
+         * @private
```

Comment:
By this, I mean, options.boundarySensitivity.  I would figure this._boundarySensitivity and DEFAULT_BOUNDARY_SENSITIVITY could just right that jsdoc train.